src/test/java/it/pagopa/transactions/commands/handlers/TransactionSendClosureHandlerTest.java

```diff
@@ -1647,6 +1684,598 @@ void shouldEnqueueErrorEventOnNodoRecoverableFailure() {
                 );
     }
 
+    @Test
+    void shouldSendClosedEventOnQueueForAuthorizedTransactionAndNodoClosePaymentResponseOutcomeKO() {
+        PaymentToken paymentToken = new PaymentToken("paymentToken");
+        RptId rptId = new RptId("77777777777111111111111111111");
+        TransactionDescription description = new TransactionDescription("description");
+        TransactionAmount amount = new TransactionAmount(100);
+        Email email = new Email("foo@example.com");
+        String faultCode = "faultCode";
+        String faultCodeString = "faultCodeString";
+        List<it.pagopa.ecommerce.commons.documents.v1.PaymentNotice> PaymentNotices = List.of(
+                new it.pagopa.ecommerce.commons.documents.v1.PaymentNotice(
+                        paymentToken.value(),
+                        rptId.value(),
+                        description.value(),
+                        amount.value(),
+                        null
+                )
+        );
+
+        TransactionActivatedEvent transactionActivatedEvent = new TransactionActivatedEvent(
+                transactionId.value().toString(),
+                new TransactionActivatedData(
+                        email.value(),
+                        PaymentNotices,
+                        faultCode,
+                        faultCodeString,
+                        Transaction.ClientId.CHECKOUT
+                )
+        );
```

Comment:
Can you use utilities from `TransactionTestUtils`?

Reply:
Fixed in https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/148/commits/e8807b65afebce1c60687eab81d4e543b0efadf7